Makefile

```diff
@@ -45,10 +45,10 @@ build-tf2-cpu:
 .PHONY: build-tf1-gpu
 build-tf1-gpu:
 	docker build -f Dockerfile.gpu \
-		--build-arg BASE_IMAGE="nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04" \
+		--build-arg BASE_IMAGE="nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04" \
 		--build-arg TENSORFLOW_PIP="tensorflow-gpu==1.15.5" \
-		--build-arg TORCH_PIP="torch==1.7.1+cu101 -f https://download.pytorch.org/whl/torch_stable.html" \
-		--build-arg TORCHVISION_PIP="torchvision==0.8.2+cu101 -f https://download.pytorch.org/whl/torch_stable.html" \
+		--build-arg TORCH_PIP="torch==1.7.0 -f https://download.pytorch.org/whl/torch_stable.html" \
+		--build-arg TORCHVISION_PIP="torchvision==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html" \
```

Comment:
is this meant to be in the same commit?

Reply:
Nope - I pushed too many commits out of my local fork. Fixed.